app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_post_comment_counter
+
```

Comment:
- [ ] you don't have a method name **update_post_comment_counter** but you have a method named **post_comments** so you can put it here :smile: as shown below :+1:

`after_save :comments_counter`